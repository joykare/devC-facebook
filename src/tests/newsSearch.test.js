import React from "react";
import { shallow } from "enzyme";
import shallowToJSON from "enzyme-to-json";
import NewsSearch from "../components/newsSearch";

describe("NewsSearch stateless component", () => {
  it("snapshot test of the component", () => {
    const component = shallow(<NewsSearch sources={[]}/>);
    let tree = shallowToJSON(component);
    expect(tree).toMatchSnapshot();
  })
})