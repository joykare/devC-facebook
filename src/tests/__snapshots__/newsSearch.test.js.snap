exports[`NewsSearch stateless component snapshot test of the component 1`] = `
<div
  className="jumbotron col-md-8 offset-md-2"
  style={
    Object {
      "backgroundColor": "transparent",
    }
  }>
  <h1
    className="display-3">
    News Sources
  </h1>
  <hr />
</div>
`;
